index.bs

```diff
@@ -1874,15 +1883,63 @@ This document mostly covers API design for the Web,
 but those who design APIs are hopefully also writing specifications
 for the APIs that they design.
 
-<h3 id="writing-resources">Other resources</h3>
-
-Some useful advice on how to write specifications is available elsewhere:
-  * <a href="https://ln.hixie.ch/?start=1140242962&amp;count=1">Writing
-    specifications: Kinds of statements</a> (Ian Hickson, 2006)
-  * <a href="https://www.w3.org/TR/qaframe-spec/">QA Framework:
-    Specification Guidelines</a> (W3C QA Working Group, 2005)
+<h3 id="requirements-on-authors-and-implementers">Identify the audience of each requirement in your specification</h3>
+<!-- was "Conformance for Documents and Implementations" in the HTML Design Principles -->
+
+The web platform is,
+unusually,
+designed to be robust in accepting poorly formed markup.
+
+When writing specifications for the web,
+be sure to note authoring requirements
+for well formed markup (the "conforming language"),
+and also
+implementation requirements
+for handling poorly formed markup (the "supported language").
+
+This allows us to
```

Comment:
Is this paragraph still necessary? Could we link out to an explanation of "robustness" in the first paragraph instead?

Reply:
I think it's a good paragraph, but we could replace it with a reference to something, sure. Do you have something in mind?